po/de.po

```diff
@@ -13091,19 +13091,19 @@ msgstr "Version: %s,%s Erstellt: %s%s"
 #~ msgstr "Einheitenfarbe (MP)"
 
 #~ msgid "VTOL HPV Cannon Bug VTOL"
-#~ msgstr "VTOL-Hochgeschwindigkeitsgeschütz Bug VTOL"
+#~ msgstr "VTOL-Hochgeschwindigkeitskanone Bug VTOL"
```

Comment:
Lines beginning with #~ contain only obsolete entries.
These are remnants of messages that no longer exist in the code,
generated by `msgmerge` and ignored by most PO editors and translators.
They don't need to be changed.